HPXMLtoOpenStudio/measure.rb

```diff
@@ -1480,7 +1480,7 @@ def self.add_heating_system(runner, model, spaces, airloop_map)
       next unless heating_system.is_heat_pump_backup_system
 
       # Store OS object for later use
-      equipment_list = model.getZoneHVACEquipmentLists.select { |el| el.thermalZone == living_zone }[0]
+      equipment_list = model.getZoneHVACEquipmentLists.find { |el| el.thermalZone == living_zone }
```

Comment:
`xxx.find` 👍

Reply:
😄 I just found out about it when @nmerket used it in his PR.

Reply:
That's the benefit of not knowing ruby well is that I have to google how to do everything...